### PR TITLE
DHFPROD-2912: Test and UI changes for 4.3.2

### PR DESCRIPTION
**Pull request by bsrikan**
This PR has:
Added some test for the back ported ES changes.
PageObject for baseUri field to add tests in the future.
Changes to QuickStart to make baseUri a required property

### CODE REVIEW

File: quick-start/src/main/ui/app/entities/entity.model.ts [hunk 1]
```diff
@@ -160,7 +160,7 @@ export class Entity {
         height: 100
       },
       'info': {
-        'baseUri': null,
+        'baseUri': 'http://example.org',
```
Comment (aebadirad) [inline quick-start/src/main/ui/app/entities/entity.model.ts]: http://example.org/, as we're doing this check:

readonly BASE_URI_REGEX = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]/;

Comment (bsrikan) [reply quick-start/src/main/ui/app/entities/entity.model.ts]: I changed it a little bit compared to what we have in 5x. I removed the last bit that expects trailing /
`+(\/)$`

Comment (aebadirad) [reply quick-start/src/main/ui/app/entities/entity.model.ts]: oh, for some reason I put that in. Whoops.

**Comment by aebadirad** [comment]
Thanks for fixing that :)

**Pull request merged**